### PR TITLE
Hueristfixes batch A

### LINKED ISSUE #1755: The results table header should be cleaner (&misc other things)
Search-results table header:

**See pr #1769 for updated checklist …**
- [ ] The hollow "sort" triangles add clutter: remove or redesign (remove and add up/down arrows on hover?)
- [ ] Add a period after "Min" in the minimum education column
- [ ] The "Experience" header makes this column far wider than its (2-digit max) data require: shorten to "Exper."
- [ ] Per the [18F content guide](https://content-guide.18f.gov/capitalization/#headings), capitalize only the first word of multi-word labels
- [ ] Consider another column-highlight technique which doesn't create a plaid effect
- [ ] Labels shouldn't break onto two lines; fix "Contract #"
- [ ] The table is too wide for the page

### LINKED ISSUE #1757: Change hover text on exclusion hover
Currently, the X next to a row of data in the explorer table only excludes that row, but the text says, "Exclude [labor category] from your search." While we do user testing to determine desired behavior, change text to say, "Exclude this row from your search" so it's at least accurate.
screen shot 2018-04-12 at 2 33 19 pm

### LINKED ISSUE #1763: "Return to the top" should be right-aligned with the content
This function at the bottom of the explorer results would be more obvious if shifted to the right edge of the content area:

### PR DESCRIPTION
**Pull request by ericronne**
This will cover a small collection of minor, usability-related fixes.

- [x] Closes #1757: Changes the hover on search results to accurately reflect what happens when the user clicks the `x`: that row (and that row only) is deleted

image

- [x] Closes #1763: "Return to the top" should be right-aligned with the content

- [x] Closes #1755:
  - [x] The hollow "sort" triangles add clutter: remove or redesign (remove and add up/down arrows on hover?)
  - [ ] ~Add a period after "Min" in the minimum education column~ Not doing, because …
  - [x] Change "Min education" to "Education" (Brings it into grammatical alignment with "Experience," which is also considered "minimum." Both get the "minimum" modifier on hover.)
  - [x] The "Experience" header makes this column far wider than its (2-digit max) data require: shorten to "Exper."
  - [x] Per the [18F content guide](https://content-guide.18f.gov/capitalization/#headings), capitalize only the first word of multi-word labels
  - [ ] ~Consider another column-highlight technique which doesn't create a plaid effect~ Later
  - [x] Labels shouldn't break onto two lines; fix "Contract #" **Note: i've changed this label to simply, "Contract". LMK if this seems confusing / wrong.**
  - [x] The table is too wide for the page **This issue has been alleviated, but the column widths fluxuate based on content. That seems odd. Will create issue.**
  - [x] Add rules above and below the header, so it provides a better visual anchor (to be recolored once #1711 is merged)
  - [x] Tabular content should be top aligned (currently bottom)

### CODE REVIEW

**Comment by hbillings** [comment]
@ericronne This should have a list of checkmarks of things you want to fix in the final PR so I know when to review it!

**Comment by ericronne** [comment]
it will. why are you chiding future me!

**Comment by ericronne** [comment]
& lists are in issues

**Comment by hbillings** [comment]
As a fellow dev, I'd like to know what heuristic fixes are or will be in "batch A" when I look at the PR. We should associate the issues. The reason for doing this up front, rather than once you're done, is to help the PR from getting too big and unwieldy. Otherwise it's easy to shoehorn lots of other things in that maybe shouldn't necessarily be there.

**Comment by ericronne** [comment]
okedoke, _hold pls …_

**Comment by ericronne** [comment]
Ok @hbillings this might be a good chunk to merge

**Comment by hbillings** [comment]
I think the padding on the tables might need to be changed now that we're doing top alignment instead of bottom alignment:
screen shot 2018-04-17 at 9 03 26 am

File: frontend/source/js/data-explorer/components/results-table/index.jsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ export class ResultsTable extends React.Component {
         </tbody>
         <tfoot>
           <tr>
-            <td colSpan={COLUMNS.length}>
+            <td colSpan={COLUMNS.length} className="return-link">
```
[the remark this answers is not in the document]
Comment (ericronne) [reply frontend/source/js/data-explorer/components/results-table/index.jsx]: yep

**Comment by ericronne** [comment]
@hbillings yeah i couldn't figure out how to mod that bottom padding: halp

**Comment by hbillings** [comment]
@ericronne Looks like it's from `_tables.scss`:
```
.sortable {
  td,
  td:first-child,
  td:last-child,
  th,
  th:first-child,
  th:last-child {
    position: relative;
    padding: $space-2x $space-1x $space-half;
    vertical-align: top;
  }
}
```

**Comment by ericronne** [comment]
@hbillings i think this is a good chunk.
if it's a good chunk that looks good to you …

note: the table changes will impact uploads, so we should figure out how to QA

**Pull request merged**